API_CHANGES.md

```diff
@@ -0,0 +1,8 @@
+# 10.1
```

Comment:
I think the breaking changes listed below are applicable to 10.0.3 already?

Reply:
Nope, for 10.0.3 I plan to cherry-pick the fixes into a separate branch for the release